vscode/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## [1.3.0] - 2022-09-05
+
+- updated `sql-formatter` to v10.x
+- updated to Nearley parser
```

Comment:
IMHO there's no point in listing internal sql-formatter changes which don't effect VSCode extension in any way. Like the switch to Nearly parser required no changes in the VSCode extension.

It would be more useful though to mention the exact version name of sql-formatter.

Reply:
Added this as a note for potential performance impact